viam-cartographer_test.go

```diff
@@ -627,6 +625,98 @@ func TestSLAMProcess(t *testing.T) {
 	internaltesthelper.ClearDirectory(t, dataDir)
 }
 
+func TestClose(t *testing.T) {
+	logger := golog.NewTestLogger(t)
+	ctx := context.Background()
+
+	t.Run("is idempotent and makes all endpoints return closed errors", func(t *testing.T) {
+		dataDir, err := internaltesthelper.CreateTempFolderArchitecture(logger)
+		test.That(t, err, test.ShouldBeNil)
+
+		grpcServer, port := setupTestGRPCServer(t)
+		test.That(t, err, test.ShouldBeNil)
+		attrCfg := &vcConfig.Config{
+			Sensors:       []string{},
+			ConfigParams:  map[string]string{"mode": "2d"},
+			DataDirectory: dataDir,
+			Port:          "localhost:" + strconv.Itoa(port),
+			UseLiveData:   &_false,
+		}
+
+		svc, err := internaltesthelper.CreateSLAMService(t, attrCfg, logger, false, testExecutableName)
+		test.That(t, err, test.ShouldBeNil)
+
+		grpcServer.Stop()
+		test.That(t, svc.Close(context.Background()), test.ShouldBeNil)
+		test.That(t, svc.Close(context.Background()), test.ShouldBeNil)
```

Comment:
is this duplicate line to make sure that the second time `svc.Close`  is called, nil is returned?

Reply:
yes

Reply:
Should we add it to a comment? It took me a second to realize it was not a typo

Reply:
done